### PR TITLE
feat: upgrade TTK version with breaking changes

### PR DESCRIPTION
**Pull request by vijayg10**

### CODE REVIEW

File: ml-testing-toolkit-cli/values.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 image:
-  repository: mojaloop/ml-testing-toolkit
-  tag: v14.0.4
+  repository: mojaloop/ml-testing-toolkit-client-lib
```
Comment (mdebarros) [inline ml-testing-toolkit-cli/values.yaml:6]: Nothing we necessarily need to change right now, but this is not the greatest name for a docker image.

Lib does not imply a CLI or something like that, I feel like we name much better, i.e.

```
  repository: mojaloop/ml-testing-toolkit-client-cli
```

Something to consider in future.

**Review by mdebarros — COMMENTED** [review]
PR is looking good....

Just a single comment that we can address/discuss at a later point.

File: ml-testing-toolkit-cli/values.yaml (hunk 1 above)
Comment (vijayg10) [reply ml-testing-toolkit-cli/values.yaml:6]: Sure

**Review by mdebarros — APPROVED** [review]
+1

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**